ethcore/sync/src/chain/supplier.rs

```diff
@@ -232,6 +219,24 @@ impl SyncSupplier {
 		Ok(Some((BlockHeadersPacket.id(), rlp)))
 	}
 
+	/// Respond to GetPooledTransactions request
+	fn return_pooled_transactions(io: &dyn SyncIo, r: &Rlp, peer_id: PeerId) -> RlpResponseResult {
+		const LIMIT: usize = 256;
```

Comment:
We already have a bunch of constants like this, we should keep them nearby with proper documentation.